src/main/java/io/cdap/e2e/pages/locators/WranglerLocators.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.locators;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * CDF wrangler related locators.
+ */
+public class WranglerLocators {
+  @FindBy(how = How.XPATH, using = "//div[@data-testid='common-components-scrollableList-scrollDownButton']")
```

Comment:
relying on DOM structure (i.e. the `div` selector) is not ideal. (What if we rewrite this UI element as a `button` or a `span` or a `li` ?). We should rely only on the unique XPATHs based on data-testids.

The data-testid based XPATHs can be copied directly using the latest version of the testids extension. 

<img width="1512" alt="Screenshot 2024-10-23 at 9 28 57 AM" src="https://github.com/user-attachments/assets/6afdf5c5-9b62-46db-8cd4-92381d8b68d1">

Reply:
done